core/pkg/registryadaptors/GCP/v1/gcpadaptorutils.go

```diff
@@ -0,0 +1,48 @@
+package v1
+
+import (
+	"fmt"
+
+	"github.com/kubescape/kubescape/v2/core/pkg/registryadaptors/registryvulnerabilities"
+	grafeaspb "google.golang.org/genproto/googleapis/grafeas/v1"
+)
+
+func (GCPAdaptor *GCPAdaptor) getImageLastScanId(imageID *registryvulnerabilities.ContainerImageIdentifier) (string, error) {
+	var s string
```

Comment:
What is this function used for?

Reply:
its a todo thing still not figured out if we have image last scan id kind of thing in gcp however the interface had a function so I added this, I can add `//todo` as a comment and maybe just return a " " inplace of string s